shared/dataRepository.ts

```diff
@@ -39,3 +47,9 @@ const generateFilePath = (time: NzTimeIso) => {
   const date = DateTime.fromISO(time).toFormat("yyyy-LL-dd");
   return `${CONSTANTS.dataFolder}/${date}.txt`;
 };
+
+const getLatestVaccineDataFilePath = async () => {
+  const files = await fs.readdir(CONSTANTS.dataFolder);
+  const latestDataFile = files.sort()[files.length - 1];
```

Comment:
Can we rely on this default `.sort`? Actually I don't know

Reply:
I named the files like `2021-10-30.txt`. So as long as the sort is a standard string sort. I think we should be fine right?

Reply:
Yeah I think so too